src/back/wgsl/writer.rs

```diff
@@ -802,6 +1071,25 @@ impl<W: Write> Writer<W> {
     }
 }
 
+fn is_builtin_supported(built_in: crate::BuiltIn) -> bool {
```

Comment:
why aren't we using `builtin_str(built_in).is_some()` instead of having this separate function?